.circleci/config.yml

```diff
@@ -18,20 +17,21 @@ jobs:
                   exit 0
               fi
 
-              OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-${OTP_VERSION}.tar.gz"
-
               set -xe
-              curl -fSL -o otp-src.tar.gz "$OTP_DOWNLOAD_URL"
-              echo "$OTP_DOWNLOAD_SHA256  otp-src.tar.gz" | sha256sum -c -
-              mkdir -vp ~/src/otp
-              tar -xzf otp-src.tar.gz -C ~/src/otp --strip-components=1
-              rm otp-src.tar.gz
+              ( mkdir -p ~/src && cd ~/src \
+                && git clone --branch maint-21 https://github.com/erlang/otp.git )
               ( cd ~/src/otp \
+                && git checkout f7012b7a731924193cf05fc77b103e89bd0fcbfb \
```

Comment:
Please see dcos's build steps for more information. https://github.com/dcos/dcos/blob/d92bb57c85f885114b1f5fd7dceb6c42b13646f9/packages/erlang/build#L10-L21